dice.js

```diff
@@ -0,0 +1,30 @@
+/*
+    - Create a file in this repo and name it `dice.js`
+    - Write a function that chooses two random numbers (1-6) and returns them. It should also return the sum.
+    - Example: `rollDice() // Dice rolled are 6 and 1.  Sum is 7`
+    
+    - BONUS: have your function take an argument of the # of dice to be rolled.
+    - Example: `rollDice(3) // Dice rolled are 4, 2, and 7.  Sum is 13.`
+
+*/
+
+let rollDice = function (no_of_dice) {
```

Comment:
a shorter way of doing this
`
const rollDice = function (numberOfRolls) {
      let stre ="Dice rolled are ";
      let sum=0;
      let diceArray = [];
      for(i=0;i<numberOfRolls;i++)
        {
          diceArray.push(Math.floor(Math.random()*6)+1);
          sum+=diceArray[i];
        }
      let lastElement=diceArray.pop(); 
      stre+=diceArray.join();
      console.log(stre+" and "+lastElement+". Sum is "+sum+".");
`